river/proba/gaussian.py

```diff
@@ -274,7 +273,7 @@ def __call__(self, x):
         var = self.var
         if var is not None:
             try:
-                return multivariate_normal(self.mu, var).pdf(x)
+                return multivariate_normal([*self.mu.values()], var).pdf(x)
```

Comment:
There might be a bug here. You have to align `self.mu` with `x`. For instance:

```py
return multivariate_normal([self.mu[i] for i in x], var).pdf(x)
```

See what I mean? Same goes `.cdf(x)`

Reply:
Thank you for pointing out this. I realized that neither mu and var were sorted the same way. Made both sort their keys alphabetically. Now I let x adapt the order of keys, which is consistent between mu and var.